### DISCUSSION TITLE
نام کانفیگ ؟

### DISCUSSION THREAD

**Discussion by BehnamAB** [hiddify/Hiddify-Manager#245] [Q&A سوالات رایج]
سلام میخواستم بدونم میشه کاری کرد که نام کانفیگها رو تغییر داد از داخل پنل که وقتی لینک ساب میدیم چیزی نشون داده شه که ما میخایم ؟ نه اینطوری

tls_grpc_CDN_vmess_xxxxxxxxxxx

**Comment by zZedix** [comment]
خواهش میکنم یا کاری کنید اسم کانکشن هارو خودمون انتخاب کنیم ! یا حداقل اسم دامنه رو از روش حذف کنید

**Comment by javad490** [comment]
دقیقا درست گفتید
چون وقتی اسم دامنه داخل لینک کانفیگ ها باشه زود باعث فیلتر و مسدود شدن اون دامنه میشه !!

**Comment by hiddify-com** [comment]
اون نام هیچ جا ثبت نمیشه فقط واسه تشخیص راحت تره برای کاربران فیلترشدن ربطی
به این موضوع نداره

**Comment by akv79** [comment]
آخر هر کانفیگ هرچی که بعد از هشتگ نوشته شده میشه اون اسم نمایشی
به عنوان مثال:
trojan://bd8e50ef-600e-9832-8b6f-2f1719f8fe9a@domain.com:443?security=tls&sni=domain.com&type=ws&path=/3NZ3PyUCgIdUj2Jfxtr3eRsPRZQ/trojanws&host=domain.com&fp=chrome&headerType=None#**Har-Chi-Ke-Delet-Mikhad**

اگر هم QR میدی به کاربرات اول خودت آدرس کانفیگ مورد نظرت رو کپی کن بعد اون آخرش رو تغییر بده بعدش از یه QR Generator آنلاین برای تولید QR استفاده کن

**Comment by javad490** [comment]
منظورم اینه که با دیدن نام دامنه در لینک کانفیگها ممکنه بعضی از کاربران یا دوستان اونها دامنه را بعنوان فیلترشکن گزارش بده و مسدودش کنندداخل مردم کمک خبرچین نداریم و همه فیلترینگ هم فقط سیستمی نیست

Sent from Yahoo Mail on Android 
 
  On Sat, Mar 4, 2023 at 12:53, ***@***.***> wrote:   


آخر هر کانفیگ هرچی که بعد از هشتگ نوشته شده میشه اون اسم نمایشی
به عنوان مثال:
***@***.***:443?security=tls&sni=domain.com&type=ws&path=/3NZ3PyUCgIdUj2Jfxtr3eRsPRZQ/trojanws&host=domain.com&fp=chrome&headerType=None**#Har-Chi-Ke-Delet-Mikhad**

اگر هم QR میدی به کاربرات اول خودت این آدرس رو کپی کن اون آخرش رو تغییر بده بعدش از یه QR Generator آنلاین برای تولید QR استفاده کن
